### PR TITLE
Added video captions.

### PR DESCRIPTION
**Pull request by dblock**
Companion to https://github.com/artsy/positron/pull/1007, displays video captions when available.

screen shot 2017-04-07 at 11 34 19 am

I am not sure how to test AMP.

### CODE REVIEW

File: desktop/apps/article/templates/amp_mixins.jade [hunk 1]
```diff
@@ -61,6 +61,9 @@ mixin amp_video(section)
       amp-youtube(data-videoid=embed.getId(section.url) layout="responsive" width="480" height="270")
     else if section.url.match('vimeo')
       amp-vimeo(data-videoid=embed.getId(section.url) layout="responsive" width="480" height="270")
```
Comment (dblock) [inline desktop/apps/article/templates/amp_mixins.jade]: I noticed we exclude any other video than youtube and vimeo?

Comment (eessex) [inline desktop/apps/article/templates/amp_mixins.jade]: Yeah for the moment we only support embeds from these providers. Are there other video hosts you think we should add?

**Comment by eessex** [comment]
@kanaabe will know better about getting these ready for amp. 

We should also include styling for these, should match the image captions -- https://github.com/artsy/force/blob/master/desktop/components/article/stylesheets/image.styl#L48

**Comment by kanaabe** [comment]
you can add /amp to the end of an article url to see the amp version @dblock

**Comment by dblock** [comment]
I tried https://www.artsy.net/article/artsy-editorial-why-are-there-so-many-great-women-video-artists/amp and that 404s, I can see the code though.

**Comment by dblock** [comment]
Added styling to captions, thanks for catching @eessex.

screen shot 2017-04-07 at 12 38 18 pm

File: desktop/apps/article/templates/amp_mixins.jade (hunk 1 above)
Comment (dblock) [inline desktop/apps/article/templates/amp_mixins.jade]: I would audit whether there're other links that people post, otherwise not anyone I can think of.

**Pull request merged**